lib/tuya.js

```diff
@@ -478,6 +505,12 @@ const msLookups = {
     },
 };
 
+const tvThermostatMode = {
+    0: 'auto',
+    1: 'manual',
+    3: 'holiday',
```

Comment:
Are you sure this shouldn't have been 2: 'holiday'? Is this maybe causing https://github.com/Koenkk/zigbee2mqtt/issues/16648 ?